src/main/java/net/openhft/chronicle/bytes/internal/ByteBuffers.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2016-2022 chronicle.software
+ *
+ *       https://chronicle.software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.openhft.chronicle.bytes.internal;
+
+import net.openhft.chronicle.core.Jvm;
+
+import java.lang.reflect.Field;
+import java.nio.ByteBuffer;
+
+/**
+ * Utility class for working with {@link ByteBuffer} instances.
+ * <p>
+ * Provides functionality to directly set the address and capacity of a ByteBuffer.
+ * <p>
+ * Note: This class is deprecated and will be removed in version x.26.
+ */
+public final class ByteBuffers {
```

Comment:
If this class is deprecated, can we add a deprecated annotation.

Also `@deprecated` in the javadoc.

Reply:
The original was deprecated, but not this one.